engines/base/pom.xml

```diff
@@ -97,6 +97,19 @@
             <version>4.2.0</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>testcontainers</artifactId>
+            <version>1.19.1</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>1.19.1</version>
```

Comment:
`<scope>test</scope>` seems to be missing here

Reply:
Added missing scope

Reply:
Update: after adding the scope all tests has failed: `TestEngine with ID 'junit-jupiter' failed to discover tests`